src/client/render/cubemap.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+Minetest
+Copyright (C) 2019 srifqi, Muhammad Rifqi Priyo Susanto
+		<muhammadrifqipriyosusanto@gmail.com>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#include "cubemap.h"
+#include <ICameraSceneNode.h>
+#include "client/camera.h"
+#include "client/hud.h"
+
+RenderingCoreCubeMap::RenderingCoreCubeMap(
+	IrrlichtDevice *_device, Client *_client, Hud *_hud)
+	: RenderingCore(_device, _client, _hud)
+{}
+
+void RenderingCoreCubeMap::initTextures()
+{
+	u32 size = screensize.Y / 3;
+	core::dimension2du sz = {size, size};
+	initTextures(sz);
+}
+
+void RenderingCoreCubeMap::initTextures(core::dimension2du size)
+{
+	image_size = {screensize.X, screensize.Y};
+	render_size = size;
+
+	for (int i = 0; i < 6; i ++)
```

Comment:
constant instead of raw 6 please

Reply:
`faceCount` (?)

Reply:
`RenderingCoreCubeMap::face_count` is used.